python/helios/platforms.py

```diff
@@ -64,6 +64,7 @@ class StaticPlatformSettings(PlatformSettingsBase):
     x: float = 0
     y: float = 0
     z: float = 0
+    is_on_ground = False
```

Comment:
The property for `ScenePart` is called `force_on_ground`. As the two things do the same, they should be consistent with each other?